sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/AIServices.java

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package com.azure.search.documents.indexes.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.Generated;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * Parameters for Azure Blob Storage knowledge source.
```

Comment:
This description seems off, but this is also coming from the Swagger. Should make sure the model in Swagger has the right description.